README-idoverrideuser.md

```diff
@@ -0,0 +1,442 @@
+```yaml
+---
+- name: Playbook to make sure test user test_user is present in idview test_idview without gecos
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      gecos: ""
+```
+
+
+Example playbook to make sure test user test_user is present in idview test_idview with gidnumber
+
+```yaml
+---
+- name: Playbook to make sure test user test_user is present in idview test_idview with gidnumber
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      gidnumber: 20001
+```
+
+
+Example playbook to make sure test user test_user is present in idview test_idview without
+
+```yaml
+---
+- name: Playbook to make sure test user test_user is present in idview test_idview without
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      gidnumber: ""
+```
+
+
+Example playbook to make sure test user test_user is present in idview test_idview with homedir /Users
+
+```yaml
+---
+- name: Playbook to make sure test user test_user is present in idview test_idview with homedir /Users
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      homedir: /Users
+```
+
+
+Example playbook to make sure test user test_user is present in idview test_idview without homedir
+
+```yaml
+---
+- name: Playbook to make sure test user test_user is present in idview test_idview without homedir
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      homedir: ""
+```
+
+
+Example playbook to make sure test user test_user is present in idview test_idview with shell
+
+```yaml
+---
+- name: Playbook to make sure test user test_user is present in idview test_idview with shell
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      shell: /bin/someshell
+```
+
+
+Example playbook to make sure test user test_user is present in idview test_idview without shell
+
+```yaml
+---
+- name: Playbook to make sure test user test_user is present in idview test_idview without shell
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      shell: ""
+```
+
+
+Example playbook to make sure test user test_user is present in idview test_idview with sshpubkey
+
+```yaml
+---
+- name: Playbook to make sure test user test_user is present in idview test_idview with sshpubkey
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      sshpubkey:
+      - ssh-rsa AAAAB3NzaC1yc2EAAADAQABAAABgQCqmVDpEX5gnSjKuv97Ay ...
+```
+
+
+Example playbook to make sure test user test_user is present in idview test_idview without
```

Comment:
sshpubkey is missing
Example playbook to make sure test user test_user is present in idview test_idview without "sshpubkey"

Reply:
Fixed.